### ISSUE TITLE
Script Fails Silently When SSL Information is Not Provided

### ISSUE THREAD

**Issue by sunami09** [esnet/sense-rtmon#42]
When the user doesn't provide the ssl information, it still shows success! message.

**Output**
```bash
Please enter the existing certificates and key:
ssl certificate (fullchain): 
ssl certificate key (privkey): 
Please enter the domain name of this machine: 
Grafana Running port (default 3000 running behind Nginx): 
Traceback (most recent call last):
  File "certify.py", line 9, in <module>
    lines[0] = f"proxy_pass http://{str(sys.argv[1])}:{str(sys.argv[2])}/;\n"
IndexError: list index out of range
!!    Success!
!!    current ssl certificate updated in cloud/nginx/proxy_conf and cloud/nginx/server_conf
!!    grafana port updated in docker-stack.yml file (default 3000)
!!    make sure the same port is entered in config files in config_cloud/
!!    what's next?
!!    run ./start.sh to start containers
```

**install.sh**
```bash
if [ "$sslmode" == "2" ]; then # existing certificate
    echo "!!    Using existing certificates (e.g. default path /etc/pki/tls )."
    echo "Please enter the existing certificates and key:"
    read -r -p "ssl certificate (fullchain): " ssl_certificate
    read -r -p "ssl certificate key (privkey): " ssl_certificate_key
    read -r -p "Please enter the domain name of this machine: " domain
    read -r -p "Grafana Running port (default 3000 running behind Nginx): " grafana_port
    
    grafana_port=${grafana_port:-3000}
//It doesn't check if the variables are null or empty.
python3 certify.py ${domain} ${grafana_port} ${ssl_certificate} ${ssl_certificate_key}

```

This creates a error in the python script certify.py when it is trying to access _sys.argv[]_ 

**certify.py**

```python
with open("./nginx/proxy_conf") as f:
    lines = f.readlines()

lines[0] = f"proxy_pass http://{str(sys.argv[1])}:{str(sys.argv[2])}/;\n"

with open("./nginx/proxy_conf", "w") as f:
    f.writelines(lines)
    
with open("./nginx/server_conf", "w") as f:
    f.write(f'''server_name {str(sys.argv[1])};
ssl_certificate     "{str(sys.argv[3])}";
ssl_certificate_key "{str(sys.argv[4])}";\n''')
```

**What should I do?** @xi-yang 
- Create default values if the user doesn't provide that
- Exit the program with Failed Message

**Comment by sunami09** [comment]
Ok.

**Comment by sunami09** [comment]
@xi-yang created a PR from `test-sunami` -> `workflow-handle-multiple-flow`

**Comment by sunami09** [comment]
Fixed #44
